paper/paper.md

```diff
@@ -0,0 +1,174 @@
+---
+title: 'TorchSurv: A Lightweight Package for Deep Survival Analysis'
+tags:
+  - Python package
+  - Deep Survival Analysis
+  - PyTorch
+  - TorchSurv
+authors:
+  - name: Mélodie Monod
+    orcid: 0000-0001-6448-2051
+    affiliation: "1"
+  - name: Peter Krusche
+    affiliation: "1"
+  - name: Qian Cao
+    affiliation: "2"
+  - name: Berkman Sahiner
+    affiliation: "2"
+  - name: Nicholas Petrick
+    affiliation: "2"
+  - name: David Ohlssen
+    affiliation: "3"
+  - name: Thibaud Coroller
+    orcid: 0000-0001-7662-8724
+    affiliation: "3"
+  - name: Novartis Pharma AG, Switzerland
+    index: 1
+  - name: Center for Devices and Radiological Health, Food and Drug Administration, MD, USA
+    index: 2
+  - name: Novartis Pharmaceuticals Corporation, NJ, USA
+    index: 3
+date: 18 July 2024
+bibliography: paper.bib
+---
+
+# Summary
+
+`TorchSurv` (available on GitHub and PyPI) is a Python package that serves as a companion tool to perform deep survival modeling within the `PyTorch` environment [@paszke2019pytorch]. With its lightweight design, minimal input requirements, full `PyTorch` backend, and freedom from restrictive parameterizations, `TorchSurv` facilitates efficient deep survival model implementation and is particularly beneficial for high-dimensional and complex data scenarios.
```

Comment:
@melodiemonod can we just add the pypi link to the name? or the journal is asking for the specifics in the parenthesis ?